src/docs.js

```diff
@@ -4,44 +4,73 @@ let assert = require('assert');
 let fs = require('mz/fs');
 let path = require('path');
 let recursiveReadSync = require('recursive-readdir-sync');
+let zlib = require('zlib');
+let rootdir = require('app-root-dir');
 
 async function documenter(options) {
-
   options = _.defaults({}, options, {
+    tier: null,
+    menuIndex: null,
```

Comment:
Let's make this default to something like... 10? No need to set it every time unless you're concerned with the ordering of your docs.